src/cmd/csi/server/command_test.go

```diff
@@ -43,5 +43,5 @@ func TestCsiCommand(t *testing.T) {
 	assert.NoError(t, err)
 
 	// logging new line so go test can parse output correctly
-	logger.NewDTLogger().Info("")
+	logger.Factory.GetLogger("")
```

Comment:
this doesn't look like it actually logs a new line but only gets a logger instance